docs/docs/configuration/client.rst

```diff
@@ -164,7 +140,8 @@ Number of comments to reveal on clicking the "X Hidden" link.
 data-isso-avatar
 ----------------
 
-Enable or disable avatar generation.
+Enable or disable avatar generation. Ignored if gravatar is enabled on
```

Comment:
Avatars will be disabled in addition to this client setting being ignored.
Please also move the gravatar setting to deprecated settings further down.

Reply:
Fixed.